testsuite/single_node_performance.py

```diff
@@ -143,58 +143,63 @@ class RunGroupConfig:
 
 # 0-indexed
 CALIBRATED_TPS_INDEX = -1
+CALIBRATED_COUNT_INDEX = -4
+CALIBRATED_MIN_RATIO_INDEX = -3
+CALIBRATED_MAX_RATIO_INDEX = -2
 CALIBRATION_SEPARATOR = "	"
 
-# transaction_type	module_working_set_size	executor_type	min_ratio	max_ratio	median
+# transaction_type	module_working_set_size	executor_type	count	min_ratio	max_ratio	median
```

Comment:
where does "count" come from originally again? (I read the query and it seems the binary prints it?)

Reply:
nvm, it's `count(as="count")`